ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vlan-interfaces.j2

```diff
@@ -71,7 +71,48 @@
 | {{ vlan_interface }} | {{ vlan_interfaces[vlan_interface].vrf | arista.avd.default('default') }} | {{ vlan_interfaces[vlan_interface].ipv6_address | arista.avd.default('-') }} | {{ vlan_interfaces[vlan_interface].ipv6_address_virtual | arista.avd.default('-') }} | {{ vlan_interfaces[vlan_interface].ipv6_virtual_router_address | arista.avd.default('-') }} | {{ vlan_interfaces[vlan_interface].vrrp.ipv6 | arista.avd.default('-') }} | {{ vlan_interfaces[vlan_interface].ipv6_nd_ra_disabled | arista.avd.default('-') | lower }} | {{ vlan_interfaces[vlan_interface].ipv6_nd_managed_config_flag | arista.avd.default('-') | lower }} | {{ vlan_interfaces[vlan_interface].ipv6_access_group_in | arista.avd.default('-') }} | {{ vlan_interfaces[vlan_interface].ipv6_access_group_out | arista.avd.default('-') }} |
 {%         endfor %}
 {%     endif %}
+{# VRRP #}
+{%     set vlan_interfaces_vrrp_details = [] %}
+{%     for vlan_interface in vlan_interfaces | arista.avd.default([]) %}
+{%         if vlan_interfaces[vlan_interface].vrrp_ids is arista.avd.defined %}
+{%             do vlan_interfaces_vrrp_details.append(vlan_interface) %}
+{%         endif %}
+{%     endfor %}
+{%     if vlan_interfaces_vrrp_details | length > 0 %}
+
+#### VRRP Details
 
+| Interface | VRRP-ID | Priority | Advertisement Interval | Preempt | Tracked Object Name(s) | Tracked Object Action(s) | IPv4 Virtual IP | IPv6 Virtual IP |
+| --------- | ------- | -------- | ---------------------- | --------| ---------------------- | ------------------------ | --------------- | ----------------|
+{%         for vlan_interface in vlan_interfaces_vrrp_details | arista.avd.natural_sort %}
+{%             for vrid in vlan_interfaces[vlan_interface].vrrp_ids %}
```

Comment:
Need to add a defined test for ```vrid.id```. **for vrid in vlan_interfaces[vlan_interface].vrrp_ids if vrid.id is arista.avd.defined**

Reply:
thanks !!